pkg/build/types/image_configuration.go

```diff
@@ -64,6 +64,23 @@ func (ic *ImageConfiguration) Validate() error {
 		}
 	}
 
+	if ic.OSRelease.ID == "" {
+		ic.OSRelease.ID = "alpine"
+	}
+
+	if ic.OSRelease.Name == "" {
+		ic.OSRelease.Name = "apko-generated image"
+		ic.OSRelease.PrettyName = "apko-generated image"
+	}
+
+	if ic.OSRelease.VersionID == "" {
+		ic.OSRelease.VersionID = "3.999"
```

Comment:
I assume this is what you mean by sensible?

Reply:
Or things we need to make sensible…? 😂

Reply:
Things we need to make sensible, I guess.  "edge" is probably a good versionid to use here.